app/pages/transportMeans/departure/AddDepartureTransportMeansYesNoPage.scala

```diff
@@ -14,28 +14,35 @@
  * limitations under the License.
  */
 
-package pages.transportMeans
+package pages.transportMeans.departure
 
-import controllers.transportMeans.routes
-import models.{Mode, UserAnswers}
+import controllers.transportMeans.departure.routes
+import models.{Index, Mode, UserAnswers}
 import pages.QuestionPage
-import pages.sections.transportMeans.{DeparturesSection, TransportMeansSection}
+import pages.sections.transportMeans.DepartureSection
 import play.api.libs.json.JsPath
 import play.api.mvc.Call
 
 import scala.util.Try
 
-case object AddDepartureTransportMeansYesNoPage extends QuestionPage[Boolean] {
+case class AddDepartureTransportMeansYesNoPage(departureIndex: Index) extends QuestionPage[Boolean] {
```

Comment:
Not sure about this? Is this not the yes/no question for whether they want to add **any** departure transport means? i.e. no index